dio/README.md

```diff
@@ -311,10 +311,10 @@ The Options class describes the http request information and configuration. Each
   ValidateStatus validateStatus;
 
   /// Custom field that you can retrieve it later in [Interceptor]、[Transformer] and the   [Response] object.
-  Map<String, dynamic> extra;
+  Map<String, Object?> extra;
   
   /// Common query parameters
-  Map<String, dynamic /*String|Iterable<String>*/ > queryParameters;  
+  Map<String, Object? /*String|Iterable<String>*/ > queryParameters;  
```

Comment:
Is the type hint here still needed?

Reply:
I would leave it. It's just for documentation `dynamic/Object?` both give you no idea what is supported.